standards/storing-source-code.md

```diff
@@ -21,19 +21,29 @@ New repositories for products and services live in the
 [ministryofjustice organisation](https://github.com/ministryofjustice)
 on GitHub.
 
-You can use your personal GitHub account (but you should [add your MOJ
-email address to your account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/),
-and maybe [use that for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
-Ask the Digital Service Desk or a member of the webops team to add you
-to the organisation. Our GitHub organisation requires that you use
-two-factor authorisation.
-
 Repositories should be [clearly named]({{ '/standards/naming-things' | relative_url }}),
 and have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})
 and enough documentation that someone new can get started with the
 project.
 
-If the repository contains secrets rather than code, make it private
-and restrict the people who have access to it. Discuss with the
-security engineering team whether you should encrypt the contents of
-the repository.
+### GitHub.com accounts
+
+You can use your personal GitHub account (but you should [add your MOJ
+email address to your account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/),
+and maybe [use that for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
+
+You should update your GitHub account profile so it is clear who you are, such as your display name or profile picture. This helps others easily identify who has made requests, added comments or provided approvals.
```

Comment:
This isn't necessary, and requires people to increase their public information, which some people may be uncomfortable with. I'm in the process of trying to upgrade us to [GitHub Business Cloud](https://github.com/business), which will allow us to do org membership via Okta, so we won't need any of this.

Reply:
As per other comment, fair enough if Okta will entirely satisfy this purpose